core/src/main/java/io/undertow/security/impl/FormAuthenticationMechanism.java

```diff
@@ -47,6 +47,8 @@
     public static final String LOCATION_ATTRIBUTE = FormAuthenticationMechanism.class.getName() + ".LOCATION";
 
     public static final String DEFAULT_POST_LOCATION = "/j_security_check";
+    public static final String DEFAULT_PAGE = "defaultPage";
+    public static final String OVERRIDE_INITIAL = "overrideInitial";
```

Comment:
These properties should have been defined along the others in AuthenticationMechanismFactory, and following the same standard in their names, such as LOGIN_PAGE is login-page. So we would have "default-page" and "override-initial" here.

Reply:
actually it is default_page and override_initial